### ISSUE TITLE
Extra whitespace after apostrophes in chart labels / legends

### ISSUE THREAD

**Issue by mthomas-ketchbrook** [COHHIO/ryha#37]
It appears that whitespace is getting added after an apostrophe in an axis label or legend value on any `{echarts4r}` visual (this issue is *not* present in `{reactable}` tables).  See two examples below:

![image](https://user-images.githubusercontent.com/52801195/222209897-cbabe6f8-3f67-493b-a0c2-8aabe059e0bf.png)

![image](https://user-images.githubusercontent.com/52801195/222209985-b505eba9-5c11-4199-9551-ce2c811ea2ee.png)

**Comment by mthomas-ketchbrook** [comment]
I am not able to reproduce this issue in an isolated project:

```
library(echarts4r)

data.frame(
  a = c("I do know", "I don't know"),
  b = c(1, 1)
) |> 
  echarts4r::e_charts(x = a) |> 
  echarts4r::e_pie(serie = b)
```

returns this chart:

![image](https://user-images.githubusercontent.com/52801195/222213626-096bcb18-09e6-4272-8809-ba831c96d4ab.png)

with the following setup (using the same version of R and `{echarts4r}` as what's currently deployed):

```
R version 4.2.2 (2022-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 22621)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.utf8  LC_CTYPE=English_United States.utf8    LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                           LC_TIME=English_United States.utf8    

attached base packages:
[1] stats     graphics  grDevices datasets  utils     methods   base     

other attached packages:
[1] echarts4r_0.4.4

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.10       rstudioapi_0.14   magrittr_2.0.3    tidyselect_1.2.0  xtable_1.8-4      R6_2.5.1         
 [7] rlang_1.0.6       fastmap_1.1.1     fansi_1.0.4       dplyr_1.1.0       tools_4.2.2       utf8_1.2.3       
[13] cli_3.6.0         withr_2.5.0       ellipsis_0.3.2    htmltools_0.5.4   yaml_2.3.7        digest_0.6.31    
[19] tibble_3.1.8      lifecycle_1.0.3   shiny_1.7.4       later_1.3.0       htmlwidgets_1.6.1 vctrs_0.5.2      
[25] promises_1.2.0.1  mime_0.12         glue_1.6.2        compiler_4.2.2    pillar_1.8.1      generics_0.1.3  
```

**Comment by IvanM26** [comment]
It seems that the problem has to do with the character used. Even though `’` and `'` are very similar, they behave differently. Across the app `’` is used (which I assume comes from a copy pasting of values from the PDF codes). Take a look at the modified example and the returned chart:

```
library(echarts4r)

data.frame(
  a = c("I do know", "I don’t know"),
  b = c(1, 1)
) |> 
  echarts4r::e_charts(x = a) |> 
  echarts4r::e_pie(serie = b)
```
![image](https://github.com/COHHIO/ryha/assets/60263844/be01f30b-d0f4-4496-a087-4aa2442a54a3)

**Comment by IvanM26** [comment]
The issue was solved as part of #63
